common/src/main/java/org/bonitasoft/console/common/server/preferences/constants/WebBonitaConstantsUtils.java

```diff
@@ -233,8 +181,9 @@ protected void createFolderIfNecessary(final File folder) {
     }
 
     protected boolean tenantFolderExists() {
+        // Let's check in Bonita-home if the tenant folder exists, before creating one in temp dir (to avoid attacks of type DOS):
```

Comment:
Usually we talk about `DDos` Attack not `Dos`
Furthermore, I don't understand what is happening there against Ddos attacks.

Moreover, isn't `createFolderIfNecessary` buggied ? We create folders if tenant folder exists in bonita home ?